### DISCUSSION TITLE
Table with 2 columns to compare

### DISCUSSION THREAD

**Discussion by Chloe-Bo** [fbaligand/kibana-enhanced-table#306] [Q&A]
Hello, 
I am using kibana-enhanced-table. 
I would like to make a table with 3 columns. First : ip from firewall. Second : IP list (warning IP). And Third : Compare if it's the same. 
![image](https://user-images.githubusercontent.com/122780773/212637640-43d93b8c-f93c-4e97-9a03-7b2d1d6acfef.png)
My problem is that I can't have first and second column. 
Can you help me ?

**Comment by fbaligand** [comment]
Hello,

Well, basically, you can create a computed column which Formula is `col0 == col1 ? "SAME" : "NOT SAME"`, and Format is `String`.

But I'm not sure this is what you ask for.
In that case, what would you mean by "I can't have first and second column"?
